### ISSUE TITLE
Improve typing coverage (4/n)

### ISSUE THREAD

**Issue by otaj** [Lightning-AI/lightning#13445]
## 🚀 Typing coverage

Let's improve typing coverage of PyTorch Lightning together!

I'm creating a new issue in order to increase visibility. There are three older issues (#7037, #5023, #4698) which became stale over time.

### Plan

Currently, there are 55 files which are excluded from mypy checks so that our CI does not fail. These files vastly differ in difficulty in order to make the typing complete. For this reason, we are introducing difficulty estimate for each file so that community members can choose to work on the files appropriate to their skill level.

Please, comment on this issue in order to reserve a particular file to work on. Once you do so, I will edit this top comment to avoid collisions. Once you think your work is finished, please open a PR referencing this issue which:

 * removes the corresponding line from `pyproject.toml`
 * and passes `mypy` checks with the corresponding line removed. You can test it locally by running `mypy` from root directory

If you are struggling with pushing it over the finish line, open the PR anyway and someone from our team will help you to get it there. 🚀

Please note, that it can happen that you may need to edit more than just one file. This is fine, but please keep in mind, that the goal of your PR will be to make the check passing for the chosen file. Also, please note that the difficulty is just an educated guess.

For those of you who are not familiar with the process of contributing a PR, we have prepared a [simple guide](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a) that will walk you through the necessary steps. You can do it! :rocket: :muscle: 

#### Difficulty 3 of 3

- [x] `pytorch_lightning/callbacks/progress/rich_progress.py` @donlapark #14963 

### Completed

<details>

#### Difficulty 1 of 3

- [x] `pytorch_lightning/core/decorators.py` #14044
- [x] `pytorch_lightning/profilers/advanced.py` @nninept #13792 
~- [ ] `pytorch_lightning/profilers/base.py` @LeeChanHyuk #13879~
- [x] `pytorch_lightning/loggers/base.py` @JustinGoheen #13494 
- [x] `pytorch_lightning/__setup__.py` @CyprienRicque #13472 
~- [ ] `pytorch_lightning/distributed/dist.py` @puhuk #13492~
- [x] `pytorch_lightning/strategies/single_device.py` @CyprienRicque #13532 
- [x] `pytorch_lightning/trainer/optimizers.py` @gautierdag #13470
- [x] `pytorch_lightning/utilities/distributed.py` @krishnakalyan3 #13678
- [x] `pytorch_lightning/callbacks/finetuning.py` @ar90n #13516 
- [x] `pytorch_lightning/loggers/mlflow.py` @JustinGoheen ~~#13690~~ #13691
- [x] `pytorch_lightning/tuner/tuning.py` @donlapark ~~#13616~~ #13631 
- [x] `pytorch_lightning/strategies/single_tpu.py` @CyprienRicque #13534 
- [x] `pytorch_lightning/strategies/ddp2.py` @CyprienRicque #13535 
- [x] `pytorch_lightning/strategies/parallel.py` @CyprienRicque #13556 
- [x] `pytorch_lightning/loggers/csv_logs.py` @JustinGoheen #13538 
- [x] `pytorch_lightning/tuner/lr_finder.py` @donlapark #13513 #13652 
- [x] `pytorch_lightning/strategies/dp.py` @CyprienRicque #13564 
- [x] `pytorch_lightning/profilers/simple.py` @krishnakalyan3 #14103
- [x] `pytorch_lightning/strategies/sharded_spawn.py` @krishnakalyan3 #14102
- [x] `pytorch_lightning/demos/mnist_datamodule.py` @alro923 #13929 
- [x] `pytorch_lightning/demos/boring_classes.py` @krishnakalyan3 #14201
- [x] `pytorch_lightning/tuner/batch_size_scaling.py` @ar90n #13518 

#### Difficulty 2 of 3

- [x] `pytorch_lightning/loops/epoch/training_epoch_loop.py` @himkt #13555 
- [x] `pytorch_lightning/core/mixins/device_dtype_mixin.py` @krishnakalyan3 #13704 
- [x] `pytorch_lightning/loggers/comet.py` @JustinGoheen #13689 
- [x] `pytorch_lightning/loggers/tensorboard.py` @JustinGoheen #13688 
- [x] `pytorch_lightning/strategies/horovod.py` @CyprienRicque #13570 
- [x] `pytorch_lightning/callbacks/model_checkpoint.py` @BongYang #13617
- [x] `pytorch_lightning/strategies/fully_sharded.py` @BongYang #13941 
- [x] `pytorch_lightning/loggers/neptune.py` @JustinGoheen #13692 
- [x] `pytorch_lightning/utilities/meta.py` @nninept #13763 #13868
- [x] `pytorch_lightning/strategies/tpu_spawn.py` @BongYang #13813  
- [x] `pytorch_lightning/loggers/logger.py` @JustinGoheen #13541 
- [x] `pytorch_lightning/loggers/wandb.py` @gautierdag #13483
- [x] `pytorch_lightning/callbacks/stochastic_weight_avg.py` @donlapark #13685 #13860
- [x] `pytorch_lightning/strategies/strategy.py` @CyprienRicque #13519 
- [x] `pytorch_lightning/strategies/deepspeed.py` @donlapark #13832 
- [x] `pytorch_lightning/strategies/ddp_spawn.py` @donlapark #13865 
- [x] `pytorch_lightning/strategies/ipu.py` @HalestormAI #13786
- [x] `pytorch_lightning/trainer/connectors/callback_connector.py` @krishnakalyan3 #13750
- [x] `pytorch_lightning/strategies/ddp.py` @lijm1358 #13885 
- [x] `pytorch_lightning/core/saving.py` @JustinGoheen #13932 
- [x] `pytorch_lightning/callbacks/quantization.py` @krishnakalyan3 #13782 
- [x] `pytorch_lightning/strategies/sharded.py` @lijm1358 #14184 
- [x] `pytorch_lightning/core/datamodule.py` @JustinGoheen #13693

#### Difficulty 3 of 3
~- [ ] `pytorch_lightning/trainer/callback_hook.py` @JustinGoheen #13807 ~
- [x] `pytorch_lightning/core/module.py` @JustinGoheen #13603 
- [x] `pytorch_lightning/trainer/connectors/data_connector.py` @JustinGoheen #13806
- [x] `pytorch_lightning/utilities/auto_restart.py` @donlapark #13904 
- [x] `pytorch_lightning/trainer/supporters.py` @donlapark #14633 
- [x] `pytorch_lightning/profilers/pytorch.py` @krishnakalyan3 #14405 
- [x] `pytorch_lightning/utilities/data.py` @nandwalritik #13901 
- [x] `pytorch_lightning/trainer/trainer.py` ~@JustinGoheen #13810~ @BongYang #14204 

</details>

cc @borda @justusschock @awaelchli @rohitgr7 @Borda @tchaton @aniketmaurya @kingjuno @alat-rights @carmocca @akihironitta @stancld as you were all involved in previous issues

**Comment by JustinGoheen** [comment]
@otaj I can take:

- [ ] pytorch_lightning/loggers/base.py
- [ ] pytorch_lightning/loggers/csv_logs.py
- [ ] pytorch_lightning/loggers/logger.py

**Comment by carmocca** [comment]
A recommendation for contributors: Unless a file requires <10 lines of fixes, I strongly suggest you work on separate PRs per file if possible. Sometimes mypy requires you to update the typing annotations for a different file, but it does not mean you need to completely annotate all modified files.

**Comment by otaj** [comment]
@barufa Thank you very much, added you to the list!

**Comment by donlapark** [comment]
I would like to contribute as well. I can take `pytorch_lightning/tuner/lr_finder.py`.

**Comment by otaj** [comment]
@donlapark thank you as well. Added to the list:)

**Comment by puhuk** [comment]
I will take `pytorch_lightning/distributed/dist.py`

**Comment by otaj** [comment]
@puhuk Thank you very much! You were added to the list!

**Comment by Cyprien-Ricque** [comment]
Hello! I can work on:
- ```pytorch_lightning/__setup__.py```
- ```pytorch_lightning/strategies/single_device.py```
- ```pytorch_lightning/strategies/strategy.py```

**Comment by otaj** [comment]
@Cyprien-Ricque Awesome, thanks! Added your reservations to the master comment.

**Comment by gautierdag** [comment]
Hi! 
Just did a quick PR for `pytorch_lightning/trainer/optimizers.py` here: https://github.com/Lightning-AI/lightning/pull/13470

**Comment by ar90n** [comment]
Hi! I want to take the followings.

* pytorch_lightning/callbacks/finetuning.py
* pytorch_lightning/tuner/batch_size_scaling.py

**Comment by otaj** [comment]
@gautierdag , thank you! I saw it was a fairly small PR, so it doesn't matter much, but we would still prefer in the future if you reserve it as to avoid collisions with other people who might already start working on that. But anyway, thank you :)

@ar90n, you were also added to the list, thank you :)

**Comment by gautierdag** [comment]
@otaj I'll try to take on `pytorch_lightning/loggers/wandb.py`

**Comment by otaj** [comment]
@gautierdag, perfectthank you, added to the list!

**Comment by carmocca** [comment]
@JustinGoheen I would fix their return annotation. Since they are simple wrappers for a deprecation, you can just mark the inputs and outptus as Any

```diff
-def rank_zero_experiment(*args, **kwargs) -> None:  # type: ignore[no-untyped-def]
+def rank_zero_experiment(*args: Any, **kwargs: Any) -> Any:
     rank_zero_deprecation(
         "The `pytorch_lightning.loggers.base.rank_zero_experiment` is deprecated in v1.7"
         " and will be removed in v1.9. Please use `pytorch_lightning.loggers.logger.rank_zero_experiment` instead."
@@ -77,7 +78,7 @@ class DummyLogger(logger.DummyLogger):
         super().__init__(*args, **kwargs)
 
 
-def merge_dicts(*args, **kwargs) -> None:  # type: ignore[no-untyped-def]
+def merge_dicts(*args: Any, **kwargs: Any) -> Any:
     rank_zero_deprecation(
         "The `pytorch_lightning.loggers.base.merge_dicts` is deprecated in v1.7"
         " and will be removed in v1.9. Please use `pytorch_lightning.loggers.logger.merge_dicts` instead."

```

**Comment by Cyprien-Ricque** [comment]
Hello! I can work on:
- ```pytorch_lightning/strategies/single_tpu.py```
- ```pytorch_lightning/strategies/ddp2.py```
- ```pytorch_lightning/strategies/parallel.py```
- ```pytorch_lightning/strategies/sharded_spawn.py```
- ```pytorch_lightning/strategies/dp.py```

**Comment by jiny419** [comment]
Hi! I'll try to take on ```pytorch_lightning/profilers/simple.py```

**Comment by JustinGoheen** [comment]
@otaj please add me to the following:

1. pytorch_lightning/trainer/trainer.py
2. pytorch_lightning/profilers/pytorch.py

**Comment by JustinGoheen** [comment]
@awaelchli I'd like to be assigned to the following:

- pytorch_lightning/loggers/tensorboard.py
- pytorch_lightning/core/module.py

**Comment by JustinGoheen** [comment]
@awaelchli I can take core.datamodule too, can I also be assigned pytorch_lightning/core/module.py.

I'll work on core.module first, then the tensorboard logger and datamodule.

**Comment by himkt** [comment]
I want to try `pytorch_lightning/trainer/trainer.py`. 🚋

**Comment by awaelchli** [comment]
@himkt That's nice! Thanks for the help. I predict that the trainer will be very difficult right now, almost impossible due to it's entanglement with all other files related to it. It probably should be left for later. May I suggest one of the easier ones to get started, e.g. `pytorch_lightning/loops/epoch/training_epoch_loop.py`? I could be wrong, feel free to try it if you want.

**Comment by krishnakalyan3** [comment]
I would like to work on ` pytorch_lightning/utilities/distributed.py`.

**Comment by JustinGoheen** [comment]
@otaj can I be assigned the remaining loggers:

- pytorch_lightning/loggers/mlflow.py
- pytorch_lightning/loggers/comet.py
- pytorch_lightning/loggers/neptune.py

Also, the following PRs are waiting on checks, is there anything more I need to do before the remaining check can run? 

- https://github.com/Lightning-AI/lightning/pull/13541
- https://github.com/Lightning-AI/lightning/pull/13538

**Comment by otaj** [comment]
Hi everyone! I'm sorry for not being responsive last week as I took a week AFK. Thank you @awaelchli and @akihironitta for taking over a bit.

I just updated master comment, thank you everyone who asked for work and working on it :) Namely @Cyprien-Ricque @JustinGoheen @krishnakalyan3 @donlapark @himkt @jiny419 :muscle: :rocket: 

Everyone, please check whether I assigned you correctly, it's been a lot of comments :+1:

**Comment by BongYang** [comment]
@otaj 
Hi! I want to try to work `pytorch_lightning/callbacks/model_checkpoint.py`

**Comment by otaj** [comment]
@BongYang Thank you, added you to the master comment :)

**Comment by otaj** [comment]
@donlapark thank you! Added you to the list!

Regarding `mypy` command, you're absolutely right! I knew about that PR and made a mental note to update the instructions here once it gets merged, however it seems I missed it :) The instructions are now updated :)

**Comment by otaj** [comment]
@shenoynikhil98 Perfect, thank you very much! I added you to the list

**Comment by donlapark** [comment]
I'd like to do `pytorch_lightning/strategies/deepspeed.py` next!

**Comment by krishnakalyan3** [comment]
I would like work on the following next

```
 pytorch_lightning/core/mixins/device_dtype_mixin.py
 pytorch_lightning/trainer/connectors/callback_connector.py
 pytorch_lightning/callbacks/quantization.py
```

**Comment by JustinGoheen** [comment]
@otaj  can I pick up `pytorch_lightning/trainer/trainer.py` & `pytorch_lightning/utilities/data.py`

**Comment by JustinGoheen** [comment]
@shenoynikhil98 I know you are handling `saving`; can you please update `_load_from_checkpoint` to account for errors in LightningDataModule that need to be resolved in `saving`.

```sh
src/pytorch_lightning/core/datamodule.py:243: error: Argument 1 to "_load_from_checkpoint" has incompatible type "Type[LightningDataModule]"; expected "Union[LightningModule, LightningDataModule]"  [arg-type]
src/pytorch_lightning/core/datamodule.py:244: error: Argument 2 to "_load_from_checkpoint" has incompatible type "Union[Union[str, Path], IO[Any]]"; expected "Union[str, IO[Any]]"  [arg-type]
src/pytorch_lightning/core/datamodule.py:246: error: Argument "hparams_file" to "_load_from_checkpoint" has incompatible type "Union[str, Path, None]"; expected "Optional[str]"  [arg-type]
```

code that resolves is:

```python
# add Path and Type
from typing import Any, Callable, Dict, IO, MutableMapping, Optional, Path, Type, Union
```

```python
def _load_from_checkpoint(
    cls: Union["pl.LightningModule", Type["pl.LightningDataModule"]],
    checkpoint_path: Union[_PATH, IO],
    map_location: Optional[Union[Dict[str, str], str, torch.device, int, Callable]] = None,
    hparams_file: Optional[_PATH] = None,
    strict: Optional[bool] = None,
    **kwargs: Any,
) -> Any:
```

**Comment by otaj** [comment]
Hi, @donlapark, @krishnakalyan3, @JustinGoheen, thank you very much, you were added to the list.

@JustinGoheen, regarding `saving`, I'd suggest waiting for a while - you have a lot of the modules you assigned to yourself, so I believe you can have something else to work on in the meantime :)

**Comment by otaj** [comment]
Hi @HalestormAI, thank you, you have been added to the list!

**Comment by otaj** [comment]
Hi @kingjuno, @jiny419, @barufa, @Cyprien-Ricque. Thank you for interest, however you have taken up files which didn't have an issue opened within two weeks. In order to allow other members of community to work on these files, I'm unassigning you now. Namely, it's these 7 files.
```
pytorch_lightning/core/decorators.py
pytorch_lightning/profilers/advanced.py
pytorch_lightning/profilers/base.py
pytorch_lightning/profilers/simple.py
pytorch_lightning/demos/mnist_datamodule.py
pytorch_lightning/strategies/sharded_spawn.py
pytorch_lightning/demos/boring_classes.py
```

**Comment by otaj** [comment]
Hi @nninept, thank you, you were added to the list!

**Comment by otaj** [comment]
Hi, @sai130302, thank you, I just added you to the list

**Comment by nninept** [comment]
I can take `pytorch_lightning/profilers/advanced.py`

**Comment by alro923** [comment]
Hi, Can I take up `pytorch_lightning/demos/mnist_datamodule.py` ?

**Comment by nssensalo** [comment]
Hi, this would be my first contribution. I am looking at some in the 1of 3 difficulty section. May I be assigned?
![image](https://user-images.githubusercontent.com/85596471/180564903-4bfbc2a4-a553-4008-a687-69233f822488.png)

**Comment by yeonju7kim** [comment]
Hi, Can I take up pytorch_lightning/strategies/fully_sharded.py?

**Comment by BongYang** [comment]
Can i take `pytorch_lightning/strategies/tpu_spawn.py`, `pytorch_lightning/strategies/ddp_spawn.py`?

**Comment by JustinGoheen** [comment]
@otaj I opened draft PRs for my remaining .trainer/ tasks. Regarding my open logger, datamodule, and module PRs:

- #13603 will be resolved in the next couple of days
- #13693 is blocked by `pytorch_lightning/core/saving.py`
- #13692 is blocked by #13689; which in turn, 13689 is potentially blocked by #13805

Regarding the .trainer PRs: I plan to work #13806, #13807, #13809 first, and then #13810

**Comment by krshrimali** [comment]
@alro923 - Done! 🎉  Added you to the list, thanks for your interest! ⚡
@nssensalo - Done! 🎉 Added you to the list for `pytorch_lightning/strategies/sharded_spawn.py` and `pytorch_ligthning/profilers/simple.py` - `mnist_datamodule.py` has been picked up by @alro923 as we are trying to follow first come first serve. Thanks for your interest! ⚡  
@BongYang - Done! 🎉 Added you to the list corresponding to the issues you expressed interest in. Thanks! ⚡

**Comment by lijm1358** [comment]
Hi, can i take <code>pytorch_lightning/strategies/ddp.py</code>?

**Comment by krshrimali** [comment]
> Hi, can i take `pytorch_lightning/strategies/ddp.py`?

Hi, @lijm1358 - Done! 🎉 Thanks for your interest, I've added you to the list. Feel free to reach out or create a draft PR and request our review in case you are stuck. Thanks! ⚡

**Comment by otaj** [comment]
Thank you, everyone who signed up and @krshrimali for being emoji master and assigning the issues!

**Comment by LeeChanHyuk** [comment]
Hello, can i take
pytorch_lightning/profilers/base.py ?

**Comment by otaj** [comment]
Hi, @LeeChanHyuk, thank you very much, just added you to the list!

**Comment by donlapark** [comment]
Phew, that took quite a while! I would like to take `pytorch_lightning/strategies/ddp_spawn.py`.

**Comment by otaj** [comment]
Thank you, @donlapark, I just added you to the list!

**Comment by nandwalritik** [comment]
Can I take `pytorch_lightning/utilities/data.py` and `pytorch_lightning/profilers/pytorch.py` ?

**Comment by otaj** [comment]
@nandwalritik, thank you! I added you to the list :zap:

**Comment by beomwookang** [comment]
Hi! Can I take `pytorch_lightning/core/decorators.py` ?

**Comment by otaj** [comment]
@beomwookang, absolutely! I added you to the list! :muscle:

**Comment by donlapark** [comment]
Hi! I would like to take `pytorch_lightning/utilities/auto_restart.py`.

**Comment by otaj** [comment]
@donlapark, awesome! You're in the master comment now!

**Comment by HeegonJin** [comment]
Hi! Can I take pytorch_lightning/callbacks/progress/rich_progress.py, if it's available?

**Comment by otaj** [comment]
It absolutely is, @HeegonJin! Thank you, you were added to the list :zap:

**Comment by JustinGoheen** [comment]
@otaj @krishnakalyan3 @akihironitta I closed #13807 because `pytorch_lightning.trainer.callback_hook.py` will be removed after the 1.7 release.

https://github.com/Lightning-AI/lightning/pull/13807#issuecomment-1196522743

**Comment by otaj** [comment]
Hi @shenoynikhil98. Since we didn't get an open PR within two weeks, I'm unassigning you from `pytorch_lightning/core/saving.py` so as to not block other community members. Thank you very much! :zap:

**Comment by JustinGoheen** [comment]
@otaj @krishnakalyan3  @akihironitta  I opened #13932 without first being assigned based on convo in https://github.com/Lightning-AI/lightning/pull/13693#discussion_r933303922

**Comment by wonbbnote** [comment]
Can I take pytorch_lightning/core/saving.py ?

**Comment by donlapark** [comment]
The pull request for`pytorch_lightning/strategies/ddp_spawn.py` has been merged in #13865 @otaj.

**Comment by otaj** [comment]
@donlapark Thank you, seems I missed to link that one.

**Comment by awaelchli** [comment]
Small milestone reached: We completed all files under the loggers module. Thanks for the help everyone. 🚀

**Comment by otaj** [comment]
Hi @nssensalo, @CyprienRicque, @sai130302, since we didn't get an open PR within two weeks, I'm unassigning you from your files so as to not block other community members. Thank you very much! :zap: Namely, it's these 4 files:

```
pytorch_lightning/profilers/simple.py
pytorch_lightning/strategies/sharded_spawn.py
pytorch_lightning/strategies/sharded.py
pytorch_lightning/demos/boring_classes.py
```

**Comment by krishnakalyan3** [comment]
@otaj can you please assign the following to me? I will be happy to take a stab at it. 

```
pytorch_lightning/profilers/simple.py
pytorch_lightning/strategies/sharded_spawn.py
```

**Comment by otaj** [comment]
@krishnakalyan3, happy to! :tada: You were added to the master list! :rocket:

**Comment by lijm1358** [comment]
Hi, can i take `pytorch_lightning/strategies/sharded.py`?

**Comment by otaj** [comment]
Hi, @lijm1358, absolutely! :zap: You were added to the list, thank you! :rocket:

**Comment by krishnakalyan3** [comment]
I would like to take ` pytorch_lightning/demos/boring_classes.py`

**Comment by otaj** [comment]
@krishnakalyan3, awesome! :tada: Added you to the master list :zap:

**Comment by JustinGoheen** [comment]
can I be removed from pytorch_lightning/trainer/supporters.py and pytorch_lightning/trainer/trainer.py please

**Comment by HeegonJin** [comment]
Can I be released from `pytorch_lightning/callbacks/progress/rich_progress.py,` if it's available?

**Comment by otaj** [comment]
@JustinGoheen, @HeegonJin, absolutely, though we're gonna be sad about that

**Comment by nninept** [comment]
Hi, I can take `pytorch_lightning/callbacks/progress/rich_progress.py`!

**Comment by otaj** [comment]
Absolutely, @nninept, thank you! :tada: You were added to the list :zap:

**Comment by otaj** [comment]
Hi @nandwalritik , I've unassigned you from `pytorch_lightning/profilers/pytorch.py` file as there hasn't been any PR opened regarding in the last two weeks. Thank you for your contributions! :zap:

**Comment by BongYang** [comment]
Hi, I would like to take `pytorch_lightning/trainer/trainer.py`

**Comment by otaj** [comment]
Hi, @BongYang, thank you very much, you were added to the list! :zap:

**Comment by krishnakalyan3** [comment]
I would like to take up ` pytorch_lightning/profilers/pytorch.py`

**Comment by otaj** [comment]
Alright, @krishnakalyan3, assigned this to you! Thanks a lot :zap:

**Comment by donlapark** [comment]
Hi! I would like to take `pytorch_lightning/trainer/supporters.py`.

**Comment by awaelchli** [comment]
@donlapark Appreciated! Feel free to go ahead. It could be a challenging one, let us know if you need help!

**Comment by donlapark** [comment]
A `mypy` check has just failed because there are some errors in `tuner/lr_finder.py`, `trainer/connectors/callback_connector.py` and `trainer/connectors/callback_connector.py` (most of them are just `Unused "type: ignore"`). Maybe we should fix those errors as well?

**Comment by otaj** [comment]
Hi @nninept, I've unassigned you from `pytorch_lightning/callbacks/progress/rich_progress.py` file as there hasn't been any PR opened regarding in the last two weeks. Thank you for your contributions! :zap:

**Comment by donlapark** [comment]
I would like to take `rich_progress.py`.

**Comment by otaj** [comment]
Awesome, @donlapark! Thank you very much, you were added to the list! :zap:

**Comment by otaj** [comment]
:tada: We did it! :tada: We just closed last PR resolving typing in PyTorch portion of Lightning :tada: Thanks to everyone, who contributed, namely  @carmocca @nninept @JustinGoheen @CyprienRicque @gautierdag @krishnakalyan3 @ar90n @donlapark @alro923 @himkt @BongYang @HalestormAI @lijm1358 @nandwalritik :tada:

**Comment by Borda** [comment]
Shall we do the same with lightning Flow and Work... :otter:

**Comment by otaj** [comment]
I'm all for it! But that's gonna be an entirely different beast, since there's a lot more on the `Flow` and `Work` side
